src/components/VideoPlayer.vue

```diff
@@ -835,7 +835,7 @@ export default {
     line-height: 1.5em;
 }
 
-.skip-segment-button .material-icons-round {
```

Comment:
This shouldn't be changed, this affects the shaka-player CSS, and hence shouldn't be updated.